Core/Tests/Speckle.Core.Tests.Unit/Helpers/BinaryFormatter.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using Speckle.Core.Helpers;
+
+namespace Speckle.Core.Tests.Unit.Helpers;
+
+[TestFixture]
+[TestOf(nameof(BinaryFormatter))]
+public class BinaryFormatterTests
+{
+  const string JSON = /*lang=json,strict*/ """{"totalChildrenCount":0,"applicationId":null,"speckle_type":"Base","string prop":"simple test case","numerical prop":123}""";
+  const string WEIRDO = "ާ5D쬣ڍtr/_ţ:󺆌ᚤ䯆#ϻϥoÈȳ3\U0006e92cݽ\U00085ee7ꨖ$\U000c372e։v뾀Ƽچ\r\n썴ϐܔȜbG헙󳄶֚<ݝЃě\U000e6af0D\ud7a9ǦH\U000529adr\U0001ce21֔􉴨\U00019423!溉ł.\U000df40dʾښ7\r\nͭit\U0006cecaӊ󻱱l󻰨Ǎ䔚\u169dּ¶\U0004c9ff\U0008eab7ﮯ&\U000190a3囎ơЍ\U0001973a맇꾒\U00039ee8I`\U00041df1C@\U000dbeb1\r\nU赂͡󷧓¾蚉\U000d4636譄ʹ\U000924fdZ\U000aa470ုvv\U0009f2d8렾ΚD󱩧ƽ\u2bf9ꎞ\U000398f9ӏ\U000e0b14簮\U0001377f갑_N\r\nS34+\U000a19e2Սь̱峺ʏ͊寞XZ꼎亖ѱ[̝\u05ee\U0009b408庹\U00067996ޖߎ\U000633da䃔Ϲяꁫȿ";
```

Comment:
Where did this data come from, is this just random binary data, how was it generated?

Reply:
https://onlinetools.com/utf8/generate-random-utf8 for the WEIRDO text - might be overkill as it's all utf8 anyway. the JSON came from one of the current serialiser tests.

Another way of doing these tests could be to take a bunch of known Speckle objects, store the current hashes, and then ensure that this technique calculates those hashes. I'm happy to add any tests you'd like to see.

Reply:
Both these tests have been removed